spec/instant.html

```diff
@@ -526,16 +526,50 @@ <h1>ToTemporalInstant ( _item_ )</h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-parsetemporalinstant" aoid="ParseTemporalInstant">
-      <h1>ParseTemporalInstant ( _isoString_ )</h1>
+    <emu-clause id="sec-temporal-parsetemporalinstant" type="abstract operation">
+      <h1>ParseTemporalInstant (
```

Comment:
Good idea about inlining. That said, I'm inclined to align with the pattern used for almost all other Temporal types, where the parsing AO has no non-parsing logic (in this case, the interpretation of Z and the conversion to epoch nanoseconds) and the single call site inlines the non-parsing logic.

So I'll plan to inline this logic into ToTemporalInstant instead, matching the behavior of ToTemporalDateTime, ToTemporalTime, ToTemporalYearMonth, ToTemporalTimeZoneSlotValue, etc.

Reply:
That would make Instant the only type without a ParseTemporal${Type}String method in abstractops.html. If you pursue that change, I think all of those should _also_ be absorbed into the corresponding ToTemporal${Type} operations, leaving only ParseISODateTime in abstractops.html. I think that is worth doing, but to me would make more sense in a followup.

Reply:
Sorry, I wasn't being clear enough. I'm suggesting doing this:
* keeping ParseTemporalInstantString in abstractops.html, but adding a [[Z]] field to its output instead of transforming the Z case into `[[OffsetString]]: *"+00:00"*`
* Removing the single-callsite ParseTemporalInstant AO, and inlining it, as well as calculation of the offset and epochNs, into ToTemporalInstant. This matches the behavior of ToTemporalDateTime, ToTemporalTime, ToTemporalYearMonth, ToTemporalTimeZoneSlotValue, etc., all of which call the ParseTemporal${Type}String directly.

I believe that both of these are consistent with the behavior of other Temporal types.  Below is a visual explanation of what's in the latest push, showing consistency across all types.

Does this address your concern?

<img width="548" alt="image" src="https://github.com/tc39/proposal-temporal/assets/277214/6d770c12-f83f-4399-8751-ac1a058e7547">

Reply:
Cool, that works. 👍